pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -76,6 +87,14 @@ def configure_checkpoint_callbacks(self, checkpoint_callback: Union[ModelCheckpo
         if not self._trainer_has_checkpoint_callbacks() and checkpoint_callback is True:
             self.trainer.callbacks.append(ModelCheckpoint(dirpath=None, filename=None, mode='min'))
 
+    def configure_swa_callbacks(self):
+        for cb in self.trainer.callbacks:
+            if isinstance(cb, StochasticWeightAveraging):
+                return
+
+        # default learning rate is taking from PyTorch blogpost on SWA.
+        self.trainer.callbacks = [StochasticWeightAveraging(swa_lrs=0.05)] + self.trainer.callbacks
```

Comment:
should only be added if the flag is `True` right?
```suggestion
        if self._trainer._stochastic_weight_avg:
            self.trainer.callbacks = [StochasticWeightAveraging(swa_lrs=0.05)] + self.trainer.callbacks
```

Reply:
Done !